### PR TITLE
added new sponsors

### PR DESCRIPTION
**Pull request by abhigyanghosh30**

### CODE REVIEW

**Comment by abhigyanghosh30** [comment]
@abhigyanghosh30 someone please merge this

File: _data/partners.csv [hunk 1]
```diff
@@ -1,4 +1,10 @@
 partner,url,pic,current
+EA,https://www.ea.com/careers/careers-overview/hyderabad,easports.png,
+Joveo,https://www.joveo.com/,joveo.png,
+AuthBase,http://www.authbase.net/,authbase.png,
+DBS,hrrps://www.dbs.com/in/index/default.page,dbs.png,
```
Comment (GaurangTandon) [inline _data/partners.csv]: https ;)

**Review by GaurangTandon — COMMENTED** [review]
LGTM

**Review by GaurangTandon — COMMENTED** [review]
fixed typo

**Review by GaurangTandon — APPROVED** [review]

**Pull request merged**